### ISSUE TITLE
Dependencies for autokey v0.94.1

### ISSUE THREAD

**Issue by nemonein** [autokey/autokey#142]
Hello.
Thank you for your effort. I really appreciate it.

I tried to upload the newest autokey v0.94.1 to my PPA(ppa:nemonein/tailored), for Bionic(Ubuntu), but it failed.
After a few hours of struggle(;-0), I've found a way to fix it.

debian/control file should contain,
```
Source: autokey
Section: utils
Priority: optional
Maintainer: Troy C. (Ubuntu PPA) <ubuntu.troxor@xoxy.net>
Build-Depends:
	python3-all,
	python3-setuptools,
	python3-dbus,
	dh-python,
	debhelper,
	python3-xlib,
	python3-gi,
	gir1.2-gtk-3.0
```
```python3-xlib, python3-gi, gir1.2-gtk-3.0``` have to be added. Otherwise, the Ubuntu PPA's build system can not build it.

By the way, I removed autokey-qt package because it does not work any more.

One more thing I'd like to say.
In the About menu, it still shows the version number as '0.94.0'.
Could that be set to '0.94.1'?
![autokey-0 94 1](https://user-images.githubusercontent.com/5688288/39955564-feaba168-560b-11e8-9f1f-15164c95c69e.png)

**Comment by sporkwitch** [comment]
This is resolved in f2e8009db4deaad0802e042e7734f52de0ef8dd0

This is the version referenced in the PPA I set up earlier today.  Wish I'd noticed this issue before digging into it today; I had to figure out those deps by trial and error lol

**Comment by luziferius** [comment]
The updated dependencies are integrated into the master branch, since commit https://github.com/autokey/autokey/commit/ba364bd26aa99e468f560d406277169cebd31e35.
